index.html

```diff
@@ -170,6 +170,43 @@ <h2>
           readonly attribute FrozenArray&lt;GamepadButton&gt; buttons;
         };
       </pre>
+      <p>
+        Instances of {{Navigator}} are created with the internal slots described in the following table:
+      </p>
+      <table class="simple" dfn-for="Navigator" dfn-type="attribute">
+        <tr>
+          <th>Internal slot
+          <th>Initial value
+          <th>Description (non-normative)
+        <tr>
+          <td><dfn>[[\exposed]]</dfn>
+          <td>`false`
+          <td>A flag indicating that the {{Gamepad}} object has been exposed to script.
+        <tr>
+          <td><dfn>[[\axisMapping]]</dfn>
+          <td>`{}`
```

Comment:
In the internal slots, we can switch to using infra types. That gives us nicer ways of working with things. 

```suggestion
          <td>An empty [=ordered map=]. 
```

Reply:
Sounds good for these internal arrays. I've left the Navigator.gamepads list as an empty array [] since we need to expose that array to script for getGamepads. Is there some incantation for converting from infra types to Javascript types?